backend/coins/btc/handlers/handlers.go

```diff
@@ -196,7 +196,7 @@ func (handlers *Handlers) getAccountTransactions(_ *http.Request) (interface{},
 }
 
 func (handlers *Handlers) postExportTransactions(_ *http.Request) (interface{}, error) {
-	name := time.Now().Format("2006-01-02-at-15-04-05-") + handlers.account.Config().Code + "-export.csv"
+	name := time.Now().Format("2006-01-02-at-15-04-05-") + string(handlers.account.Config().Code) + "-export.csv"
```

Comment:
This would read nicer:

```
fmt.Sprintf("%s-%s-export.csv", time.Now().Format("2006-01-02-at-15-04-05"), handlers.account.Config().Code)
```

- no need for explicit `string(..code..)`
- no funny trailing `-` in time format
- easier to see how the final filename looks like

Reply:
:+1: done